src/RoboSaverVirtualModule.sol

```diff
@@ -82,34 +85,41 @@ contract RoboSaverVirtualModule {
                                       MODIFIERS
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @notice Checks whether a call is authorized to trigger top-up or exec queue txs
+    /// @notice Enforce that the function is called by the keeper only
     modifier onlyKeeper() {
         if (msg.sender != keeper) revert NotKeeper(msg.sender);
         _;
     }
 
     /*//////////////////////////////////////////////////////////////////////////
-                    EXTERNAL METHODS: TOP-UP AGENTS
+                                  EXTERNAL METHODS
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @dev Check condition and determine whether a task should be executed by Gelato.
+    /// @notice Check if there is a surplus or deficit of $EURe on the card
+    /// @return canExec Whether a transaction needs to be
```

Comment:
```suggestion
    /// @return canExec Whether a transaction needs to be executed
```

Reply:
i want to think a bit more about this, since `canExec` itself is a bit misleading. we are executing the queuing of a transaction, not actually executing. would be nice to prevent this ambiguity

Reply:
perhaps an opp to rename `canExec` to smth more align to queuing context. e.g: `canQueueTx`

Reply:
d771e65